tests/puppeteer/README.md

```diff
@@ -53,8 +53,19 @@ If you want to run all sanity tests "safely", you can use the Travis-specific co
 npm run sanity-travis
 ```
 
+## Functional tests 
+This campaign verifies that each function of the software application, operates in conformance with the functional requirements. 
```

Comment:
Functions are operating, not the campaign

```suggestion
This campaign verifies that each function of the software application operate in conformance with the functional requirements. 
```

Reply:
done thanks